examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -179,9 +178,10 @@ async function example_API_Delete(cacheClient: CacheClient) {
 }
 
 async function example_API_Increment(cacheClient: CacheClient) {
+  await cacheClient.set('test-cache', 'test-key', '10');
   const result = await cacheClient.increment('test-cache', 'test-key', 1);
   if (result instanceof CacheIncrement.Success) {
-    console.log("Key 'test-key' incremented by value 1 successfully");
+    console.log(`Key 'test-key' incremented successfully. New value in key test-key: ${result.toString()}`);
```

Comment:
is there a `value` property on that success response?  or like `valueNumber`?

Reply:
Yes, it's there.

Reply:
can we call that in the log statement instead of `toString()`?